WordPress/src/main/res/layout/reader_activity_subs.xml

```diff
@@ -64,10 +63,10 @@
 
         <ImageButton
             android:id="@+id/btn_add"
+            android:padding="24dp"
```

Comment:
Nice improvement. That image was really really small. 😱 

It looks like there is a generic `dimens` value that we can use. Is there a reason why you hardcoded `24dp` here? 

```suggestion
            android:padding="@dimen/margin_extra_medium_large"
```

Reply:
Thanks for spotting this. Thought it was changed. I am gonna look at this one more time to verify that all the behavior introduced for it is the best approach.

Reply:
I'm gonna fix it locally and make another push. With this padding here there's no need for extending the touch target since padding is utilized. So the touch target code has no effect.
The image size still remains by the way. I think it's super small but I am not sure if the designers want it increased. Wdyt?

Reply:
I'm okay with slightly increasing the touch target via padding. I think we can leave the image size itself unchanged.

Reply:
I was thinking to just let it stay with the size for now and then I could open another issue to readjust it.

Reply:
Okay great! Fixed in 26635f4